fs/littlefs/Make.defs

```diff
@@ -31,11 +31,10 @@ CSRCS += lfs.c lfs_util.c
 DEPPATH += --dep-path littlefs/littlefs
 VPATH += :littlefs/littlefs
 
-CFLAGS += -DLFS_TRACE=finfo
-CFLAGS += -DLFS_DEBUG=finfo
-CFLAGS += -DLFS_WARN=fwarn
-CFLAGS += -DLFS_ERROR=ferr
-CFLAGS += -DLFS_ASSERT=DEBUGASSERT
+CFLAGS += -DLFS_NO_ASSERT
+CFLAGS += -DLFS_NO_DEBUG
+CFLAGS += -DLFS_NO_ERROR
+CFLAGS += -DLFS_NO_WARN
```

Comment:
@xiaoxiang781216 

I think we should create a separate PR for these changes.
What do you think?

Reply:
I take another approach by utilizing LFS_CONFIG hook.